lblod/app-subsidiepunt
# Hotifx/add population count

## ID
DGS-320

## Description
This adds the missing data to make the klimaattabel work properly. This data was forgotten when subsidiepunt became a standalone app

## Type of change


## How to setup
Run the app, wait for migrations to finish

## How to test
Create a new LEKP 1.0 subsidy and go to "indienen voorstel" step. Tabel should appear
